src/calculateRentalCost.js

```diff
@@ -5,6 +5,17 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const COST_PER_DAY = 40;
+  const SHORT_DISCOUNT = 20;
+  const LONG_DISCOUNT = 50;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+
+  const COST_PER_VACATION = COST_PER_DAY * days;
+
+  if (days >= LONG_TERM) return COST_PER_VACATION - LONG_DISCOUNT;
```

Comment:
Consider using `const` for `COST_PER_VACATION` instead of `let` as the value is not being reassigned, to avoid unintentional changes.